### PR TITLE
Added RDP Plugin

### PR DESCRIPTION
**Pull request by MBeggiato**
Proposed to add the [RDP Plugin](https://github.com/MBeggiato/Flow.Launcher.Plugin.RDP) to the official list. <br>

### CODE REVIEW

**Comment by MBeggiato** [comment]
Fixed: ERROR ci/src/validator.py - json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1947 column 6 (char 104045)
With [this](https://github.com/MBeggiato/Flow.Launcher.PluginsManifest/commit/1342c81d13fc2206d2ef4ce92a3383bc83b7103f)

**Comment by VictoriousRaptor** [comment]
And please make the indentation consistent.

**Comment by MBeggiato** [comment]
> And please make the indentation consistent.

Done with [this](https://github.com/MBeggiato/Flow.Launcher.PluginsManifest/commit/f4ed04c2b953ed06f8e8dd5eb8025121d767c54d).
Thanks for the tip!

**Comment by MBeggiato** [comment]
The last issue is not related to my pr right?

**Comment by VictoriousRaptor** [comment]
> The last issue is not related to my pr right?

looks like you don't have that image/logo.png in your repo.

File: plugins.json [hunk 1]
```diff
@@ -1933,5 +1933,17 @@
...
+    },
+    {
+        "ID": "B7CDDEA47E354243BEE0434DDBAE438A",
+        "Name": "RDP",
+        "Description": "RDP plugin for Flow Launcher",
+        "Author": "MBeggiato",
+        "Version": "1.0.9",
+        "Language": "csharp",
+        "Website": "https://github.com/MBeggiato/Flow.Launcher.Plugin.RDP",
+        "UrlDownload": "https://github.com/MBeggiato/Flow.Launcher.Plugin.RDP/releases/download/v1.0.0.12/Flow.Launcher.Plugin.RDP.zip",
```
Comment (jjw24) [inline plugins.json:1945]: There is a mismatch between the version indicated and in your download link.

Could you also use Semantic Versioning in the form of X.Y.Z please.

**Review by jjw24 — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (MBeggiato) [reply plugins.json:1945]: I see! I fixed the version to match the form X.Y.Z. The latest release is now 1.0.13, just like in the plugin.json

Comment (jjw24) [reply plugins.json:1945]: A could of additional changes please:
1. The 'Version' field need to match the version used in the 'UrlDownload' field, see above.
2. Most importantly could we ask you to use GitHub Actions to publish your plugin please. The reason is packaging and uploading the zip manually is hard for us to make reasonable judgement that what's contained in the zip is the same from the repository.

You can follow this as the example, most likely just need to replace the dotnet version and the name of the plugin: https://github.com/taooceros/Flow.Plugin.WindowWalker/blob/master/.github/workflows/publish-action.yml

Comment (MBeggiato) [reply plugins.json:1945]: @jjw24 Thanks for the requested changes. I think I finaly got it all working. Had a bit of trouble with Github Actions but now it should work!

**Review by VictoriousRaptor — APPROVED** [review]

**Pull request merged**